GoogleForCreators/web-stories-wp
# Media Hotlinking: Implement CORS Proxy

## Context



## Summary

Create a basic cors proxy, using the REST API. 

## Relevant Technical Choices

Added feature flag. 

## To-do



## User-facing changes

https://user-images.githubusercontent.com/237508/135309701-6a2ec11d-4324-44fc-9071-5846c14e6ef9.mov

## Testing Instructions






This PR can be tested by following these steps:

1. Enable CORS Proxy experiment.
2. Insert hotlink image / video page. 
3. See image / video in page.  


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #9039
